api/src/main/java/org/pmiops/workbench/api/ErrorTestingController.java

```diff
@@ -0,0 +1,28 @@
+package org.pmiops.workbench.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+
+
+@RestController
+public class ErrorTestingController implements ErrorTestingApiDelegate {
+
+  @Override
+  public ResponseEntity<Void> failFourHundred() {
+    return new ResponseEntity<Void>(HttpStatus.BAD_REQUEST);
+  }
+
+  @Override
+  public ResponseEntity<Void> failFiveHundred() {
+    return new ResponseEntity<Void>(HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
If we decide to keep these endpoints, they should act both as documentation and a coding pattern. I expect no code will actually respond with 500, but instead it will be the by-product of an uncaught exception. Therefore, to produce this response, this code should throw an exception.